src/Directory.Build.props

```diff
@@ -37,5 +37,6 @@
         <NuGet_YamlDotNet>11.2.1</NuGet_YamlDotNet>
         <NuGet_IdentityModelTokenJwt>6.13.1</NuGet_IdentityModelTokenJwt>
         <NuGet_NLipsum>1.1.0</NuGet_NLipsum>
+        <NuGet_BouncyCastleNetCore>1.9.0</NuGet_BouncyCastleNetCore>
```

Comment:
Make sure to indicate the license from https://www.bouncycastle.org/licence.html in `THIRD-PARTY-NOTICES.md` file at the root of the repo. This file is automatically shipped with DevToys when we build.